modules/stanford_soe_helper_collection_views/stanford_soe_helper_collection_views.views_default.inc

```diff
@@ -10,6 +10,356 @@
+  $handler->display->display_options['relationships']['reverse_field_s_mag_issue_article_5_node']['label'] = 'Article 5';
+  /* Relationship: Entity Reference: Referencing entity */
+  $handler->display->display_options['relationships']['reverse_field_s_mag_issue_article_4_node']['id'] = 'reverse_field_s_mag_issue_article_4_node';
+  $handler->display->display_options['relationships']['reverse_field_s_mag_issue_article_4_node']['table'] = 'node';
+  $handler->display->display_options['relationships']['reverse_field_s_mag_issue_article_4_node']['field'] = 'reverse_field_s_mag_issue_article_4_node';
+  $handler->display->display_options['relationships']['reverse_field_s_mag_issue_article_4_node']['label'] = 'Article 4';
+  /* Relationship: Entity Reference: Referenced Entity */
+  $handler->display->display_options['relationships']['field_s_mag_article_collection_target_id']['id'] = 'field_s_mag_article_collection_target_id';
+  $handler->display->display_options['relationships']['field_s_mag_article_collection_target_id']['table'] = 'field_data_field_s_mag_article_collection';
+  $handler->display->display_options['relationships']['field_s_mag_article_collection_target_id']['field'] = 'field_s_mag_article_collection_target_id';
+  /* Relationship: Entity Reference: Referencing entity */
+  $handler->display->display_options['relationships']['reverse_field_s_mag_article_collection_node']['id'] = 'reverse_field_s_mag_article_collection_node';
+  $handler->display->display_options['relationships']['reverse_field_s_mag_article_collection_node']['table'] = 'node';
+  $handler->display->display_options['relationships']['reverse_field_s_mag_article_collection_node']['field'] = 'reverse_field_s_mag_article_collection_node';
+  $handler->display->display_options['relationships']['reverse_field_s_mag_article_collection_node']['relationship'] = 'field_s_mag_article_collection_target_id';
+  $handler->display->display_options['defaults']['fields'] = FALSE;
+  /* Field: Content: Featured Image */
+  $handler->display->display_options['fields']['field_s_mag_article_image']['id'] = 'field_s_mag_article_image';
+  $handler->display->display_options['fields']['field_s_mag_article_image']['table'] = 'field_data_field_s_mag_article_image';
+  $handler->display->display_options['fields']['field_s_mag_article_image']['field'] = 'field_s_mag_article_image';
+  $handler->display->display_options['fields']['field_s_mag_article_image']['label'] = '';
+  $handler->display->display_options['fields']['field_s_mag_article_image']['exclude'] = TRUE;
+  $handler->display->display_options['fields']['field_s_mag_article_image']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['field_s_mag_article_image']['click_sort_column'] = 'fid';
+  $handler->display->display_options['fields']['field_s_mag_article_image']['settings'] = array(
+    'image_style' => '6-col-banner-short',
```

Comment:
@minorwm This image style, 6-col-banner-short, is in Stanford Image Styles.  Make sure you've added that module as a dependency.

Reply:
@cjwest I have it as a dependency

Reply:
👍